### PR TITLE
v2ray-desktop: new, 2.1.7

### PR DESCRIPTION
**Pull request by Nexus01**
clash is a dependency of v2ray-desktop.

### CODE REVIEW

File: extra-network/clash/spec [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VER=1.1.0
+GITSRC=https://github.com/Dreamacro/clash
+GITCO=v$VER
```
Comment (chenx97) [inline extra-network/clash/spec]: It looks like we can use `$SRCTBL` here.

File: extra-network/v2ray-desktop/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,6 @@
+PKGNAME=v2ray-desktop
+PKGSEC=x11
+PKGDES="A cross-platform GUI proxy client that supports Shadowsocks, V2Ray, and Trojan protocols."
+PKGDEP="clash qt-5"
+BUILDDEP="git jq"
```
Comment (chenx97) [inline extra-network/v2ray-desktop/autobuild/defines]: `git` is unnecessary

File: extra-network/clash/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,6 @@
+PKGNAME=clash
+PKGSEC=net
+PKGDEP="glibc"
+BUILDDEP="git go"
```
Comment (chenx97) [inline extra-network/clash/autobuild/defines]: `git` is unnecessary

File: extra-network/v2ray-desktop/autobuild/prepare [hunk 4]
```diff
@@ -0,0 +1,2 @@
+acbs_copy_git
+git submodule update --init --recursive
```
Comment (chenx97) [inline extra-network/v2ray-desktop/autobuild/prepare]: Since `acbs` gets submodules for you after cloning, this file is unnecessary.

**Review by chenx97 — APPROVED** [review]

**Comment by MingcongBai** [comment]
Same as #2362 and #2363, please re-open a Pull Request with cleaned up Git history. We can't merge this as it is.

**Pull request closed without merging**